app/templates/components/footer-main.hbs

```diff
@@ -1,3 +1,4 @@
+<div class="mb-12">
```

Comment:
No need to wrap the div, add the class in the div below

Reply:
I think it will not be the priority class then so to make it important we should add a div otherwise it will be neglected

Reply:
in case when ui is mobile it will work fine otherwise grid margin bottom becomes high priority

Reply:
i found the solution